client/src/components/Cell/CellMain.tsx

```diff
@@ -36,12 +40,32 @@ const CellMain = ({
   const Confirm = useConfirm();
   const Alert = useAlert();
   const { brute, owner } = useBrute();
-
+  const { user } = useAuth();
+  const navigate = useNavigate();
   const xpNeededForNextLevel = useMemo(
     () => (brute ? getXPNeeded(brute.level + 1) : 0),
     [brute],
   );
-
+  const arrowProps = {
+    maxHeight: '70px', maxWidth: '70px', color: '#7d4736', cursor: 'pointer', fontSize: '150px', margin: 'auto'
+  };
+  const bruteList : string[] | undefined = user?.brutes.map((bruteDetails) => bruteDetails.name);
```

Comment:
No need to declare this variable, you can just look through the user brutes in the handler